latch_cli/types.py

```diff
@@ -0,0 +1,17 @@
+import datetime
+from dataclasses import dataclass
+
+from dataclasses_json import dataclass_json
+
+
+@dataclass_json
+@dataclass
+class LatchWorkflowConfig:
+    """Configuration for a Latch workflow."""
+
+    latch_version: str
+    """Latch version used to initialize the workflow."""
```

Comment:
```suggestion
    """Version of the Latch SDK used to initialize the workflow"""
```
don't repeat the variable name